docs/crictl.md

```diff
@@ -54,7 +54,7 @@ $ docker save k8s.gcr.io/pause-amd64:3.1 -o pause.tar
 Then use [`ctr`](https://github.com/containerd/containerd/blob/master/docs/man/ctr.1.md)
 to load the container image into the container runtime:
 ```console
-$ sudo ctr cri load pause.tar
+$ sudo ctr -n=k8s.io images import pause.tar
```

Comment:
should discuss/mention what the namespace is all about if using here

Reply:
Done